charts/kyverno-policy/templates/cluster/cluster-check-image.yaml

```diff
@@ -11,40 +6,10 @@ metadata:
     policies.kyverno.io/title: Verify Cosign image signatures against provided public key
     policies.kyverno.io/description: >-
       Verify Cosign image signatures against provided public key(s).
-    kyverno.io/kyverno-version: 1.6.0
-    policies.kyverno.io/subject: Pod
   name: check-image
 spec:
   validationFailureAction: {{ .Values.checkImagePolicy.validationFailureAction }}
   background: {{ .Values.checkImagePolicy.background }}
   webhookTimeoutSeconds: {{ .Values.checkImagePolicy.webhookTimeoutSeconds }}
   failurePolicy: {{ .Values.checkImagePolicy.failurePolicy }}
-  rules:
-  - name: check-image
-    match:
-      any:
-      - resources:
-          kinds:
-            - Pod
-          namespaces:
-            - "*"
-    exclude:
-      any:
-        - resources:
-            namespaces:
-              - sample-ns
-            names:
-              - "sample-name"
-    verifyImages:
-    - imageReferences:
-{{ .Values.checkImagePolicy.imageReferences | toYaml | indent 6 }}
-      mutateDigest: {{ .Values.checkImagePolicy.mutateDigest }}
-      verifyDigest: {{ .Values.checkImagePolicy.verifyDigest }}
-      attestors:
-      - count: 1
-        entries:
-        - keys:
-            publicKeys: |-
-{{- range $name, $key := $keys }}
-{{ $key | b64dec | trim | indent 14 }}
-{{- end }}
+  rules: {{ .Values.checkImagePolicy.rules | toYaml | nindent 4 }}
```

Comment:
Entire block of rules comes from values, which are overridden by customizations.